README.md

```diff
@@ -20,32 +20,28 @@ import (
 
 func main() {
 	// Generate token that is valid for 60 seconds using the default secret
-	var token string
-	var err error
-	if token, err = ft.GenerateToken("RmFzdGx5IFRva2VuIFRlc3Q=", 60*time.Second, base64.StdEncoding); err != nil {
-		log.Fatal("Error while generating token", err)
-	}
+	token := ft.GenerateToken("RmFzdGx5IFRva2VuIFRlc3Q=", 60*time.Second, base64.StdEncoding); err != nil {
```

Comment:
Oops forgot to remove the `; err != nil {`. Nah the functions have no errors to return so err will always be nil. I'm also proposing with the PR that we remove the (always) nil err